lib/spack/spack/package.py

```diff
@@ -983,6 +1008,124 @@ def check_paths(path_list, filetype, predicate):
                 "Install failed for %s.  Nothing was installed!" % self.name)
 
 
+    def global_license_file(self):
```

Comment:
@tgamblin: What about `global_license_file()` specifically? The Intel package needs to know the location of the license before installation. Would it be better to move it to `lib/spack/spack/hooks/licensing.py` and require the user to import that file, or to keep it in `package.py`, and maybe make it a property?